resp/get_foxsi_resp.pro

```diff
@@ -32,9 +32,36 @@ default, type, 'si'
 elements = ((erange[1]-erange[0])/bin)+1.
 energy = findgen(elements, increment=bin, start=erange[0])
 
+; If YEAR is not set then COMMON DATE will be used to know which FOXSI flight the user is working with.
+
+if not keyword_set(year) then begin
+
+        COMMON FOXSI_PARAM ; allows access to the FOXSI COMMON variables.
+
+        datefoxsi1 = 1.0679040e+09 ; FOXSI1 Launch Date
+        datefoxsi2 = 1.1342592e+09 ; FOXSI2 Launch Date
```

Comment:
I am being picky... but I would prefer to see
datefoxsi1 = anytim('2012-nov-03')

this is exactly the same output but this long number makes me nervous... It is hard to spot if it is giving the wrong answer. 

same for the following lines.

Reply:
Thanks @somusset for the suggestion. It is now implemented here: https://github.com/foxsi/foxsi-science/pull/85/commits/4640ab7b39684edcaf1e178ce98b19877ae2c194

Reply:
I've implemented this change, but I just wanted to note that, while the parameter list for FOXSI-1 provides anytim('2012-nov-03') as the date, other parts of our documentation show '2012-nov-02' as the actual launch date. If we agree that the date should instead be '2012-nov-02', should we make those changes to our code now (and update the parameter file), or will there be additional effects in other parts of the analysis code? I can submit an issue if we think there will be extensive effects from this change.

Reply:
I've merged the pull request regarding the date!